modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -53,25 +54,52 @@ data:
     prometheusK8s:
       remoteWrite:
       - url: "https://remote-write.endpoint"
-        <endpoint_authentication_credentials>
+        <endpoint_authentication_credentials> <1>
 ----
 +
-For `endpoint_authentication_credentials` substitute the credentials for the endpoint.
-Currently supported authentication methods are basic authentication (`basicAuth`) and client TLS (`tlsConfig`) authentication.
+<1> For `endpoint_authentication_credentials` substitute the authentication method and the credentials for the endpoint.
+Currently supported authentication methods are AWS Signature version 4 (`sigv4`), basic authentication (`basicAuth`),  bearer token (`Bearer`), TLS client (`tlsConfig`), and OAuth 2.0 (`oauth2`). 
++
+* AWS Signature Version 4
++
+This method uses AWS Signature Version 4 (`sigv4`) authentication to sign requests.
+This method cannot be used simultaneously with `basic_auth`, `Bearer`, or `oauth2`.
 +
-* The following example configures basic authentication:
+The following shows a `sigv4` sample configuration.
+It assumes you have already created a `sigv4` secret named `sigv4-credentials`in the `openshift-monitoring` namespace.
+Replace the sample `region`, `key`, `profile`, and `roleArn` values with your authentication credentials:
 +
 [source,yaml]
 ----
-basicAuth:
-  username:
-    <usernameSecret>
-  password:
-    <passwordSecret>
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+       - url: "https://authorization.remotewrite.com/api/write"
+        sigv4:
+          region: <AWS_region>
+          accessKey:
+            name: sigv4-credentials
+            key: <AWS_access_key>
+          secretKey:
+            name: sigv4-credentials
+            key: <secret>
+          profile: <AWS_profile_name>
+          roleArn: <AWS_role_arn>
 ----
-Substitute `<usernameSecret>` and `<passwordSecret>` accordingly. 
 +
-The following sample shows basic authentication configured with `remoteWriteAuth` for the `name` values and `user` and `password` for the `key` values. These values contain the endpoint authentication credentials:
+* basic authentication
++
+Basic authorization (`basicAuth`) sets the authorization header on every remote write request with the configured username and password.
++
+The following sample shows a `basicAuth` configuration.
+It assumes you have already set up authentication credentials for the endpoint.
```

Comment:
change to
It assumes you have already set up authentication credentials named `remoteWriteAuth` for the endpoint.